### ISSUE TITLE
CAUGHT AN EXCEPTION WHILE USING THE FLAG PACKAGE FOR FLUTTER APP

### ISSUE THREAD

**Issue by spellsuvam** [LunaGao/flag_flutter#76]
while i was trying to show the flags of countries  like brazil and afgansthan by using both the flagcodes and strings (flagcode.AF,"AF").
 i got a svg realted exception. The exception is given below:-
----------------Exception-------------------------
#6      SvgParserState.parse
package:flutter_svg/…/svg/parser_state.dart:885
<asynchronous suspension>
#7      SvgParser.parse
package:flutter_svg/parser.dart:27
<asynchronous suspension>
#8      Svg.fromSvgString
package:flutter_svg/svg.dart:139
<asynchronous suspension>
#9      Svg.svgPictureStringDecoder
package:flutter_svg/svg.dart:90
<asynchronous suspension>
Picture provider: ExactAssetPicture(name: "packages/flag/res/4x3/af.svg", bundle: null, colorFilter: null)
Picture key: AssetBundlePictureKey(bundle: PlatformAssetBundle#3ea37(), name: "packages/flag/res/4x3/af.svg", colorFilter: null, theme: SvgTheme(currentColor: null, fontSize: 14.0, xHeight: 7.0))
-------------------------------------------------------------------------------------------------------

**Comment by HarsHvA** [comment]
I am also getting the same issue. Did you find any solution?

**Comment by ByJerome** [comment]
> I am also getting the same issue. Did you find any solution?

@HarsHvA  LunaGao merged my fixes into the stable branch. I fixed like 99% of all problems. Just install the package into your pubspec.yaml as described:

"use it like this for now:"
```
flag:
  git:
    url: https://github.com/LunaGao/flag_flutter.git
    ref: stable
```

_Originally posted by @pauli2406 in https://github.com/LunaGao/flag_flutter/issues/77#issuecomment-1215888324_

**Comment by HarsHvA** [comment]
@ByJerome sure thanks I'll try that

**Comment by LunaGao** [comment]
fixed in 7.0.0 version.
![image](https://github.com/LunaGao/flag_flutter/assets/5066920/c9a47b5f-701e-41cb-b692-fc71efd23df7)
